main.tf

```diff
@@ -66,10 +66,6 @@ resource "google_redis_instance" "default" {
       rdb_snapshot_period = persistence_config.value["rdb_snapshot_period"]
     }
   }
-
-  lifecycle {
-    ignore_changes = [maintenance_schedule]
```

Comment:
Is the issue in https://github.com/terraform-google-modules/terraform-google-memorystore/pull/152 now fixed?

Reply:
@bharathkkb yes it was fixed in [4.74](https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#4740-july-18-2023)
[PR](https://github.com/hashicorp/terraform-provider-google/pull/15063)